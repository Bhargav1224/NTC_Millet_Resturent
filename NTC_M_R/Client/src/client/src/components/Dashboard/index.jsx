/* eslint-disable no-nested-ternary */
import React from 'react';

function Dashboard() {
  return (
    <div className="App">
      <h3>Welcome To Seeder</h3>
    </div>
  );
}

export default Dashboard;
