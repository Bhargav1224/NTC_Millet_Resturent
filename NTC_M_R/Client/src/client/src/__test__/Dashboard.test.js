/*eslint-disable*/
import renderer from 'react-test-renderer';
import Dashboard from '../components/Dashboard';

it('changes the class when hovered', () => {
  const component = renderer.create(
    <Dashboard />
    );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});