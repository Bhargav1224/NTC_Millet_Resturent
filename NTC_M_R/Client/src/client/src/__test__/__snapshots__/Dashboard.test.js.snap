// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`changes the class when hovered 1`] = `
<div
  className="App"
>
  <h3>
    Welcome To Seeder
  </h3>
</div>
`;
